packages/desktop-client/src/components/App.tsx

```
@@ -14,13 +14,13 @@
~import { styles } from '@actual-app/components/styles';
 import { View } from '@actual-app/components/view';
 
 import {
-  addNotification,
   loadGlobalPrefs,
   signOut,
 } from 'loot-core/client/actions';
```

ESLint

[error] Replace `⏎··loadGlobalPrefs,⏎··signOut,⏎` with `·loadGlobalPrefs,·signOut·`

(prettier/prettier)